js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/kit/TestKit.scala

```diff
@@ -147,17 +160,25 @@ final class TestKit(jsEnv: JSEnv, timeout: FiniteDuration) {
 
     (run, out, err)
   }
+
+  private def codeToInput(code: String): Seq[Input] = {
+    val p = Files.write(
+        Jimfs.newFileSystem().getPath("testScript.js"),
```

Comment:
```suggestion
        Jimfs.newFileSystem().getPath("test.js"),
```
I guess it doesn't make sense to kill this `testScript.js` anymore.